pinot-spi/src/main/java/org/apache/pinot/spi/utils/RebalanceConfigConstants.java

```diff
@@ -25,6 +25,12 @@ public class RebalanceConfigConstants {
   private RebalanceConfigConstants() {
   }
 
+  // Unique Id for rebalance
+  public static final String REBALANCE_ID = "rebalanceId";
```

Comment:
(minor) Since it is already under the scope of rebalance
```suggestion
  public static final String JOB_ID = "jobId";
```